README.md

```diff
@@ -23,7 +23,21 @@ For more information on the operations supported by the `sql:Client`, which incl
 
 For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/mysql-init-options.html).
 
-## Building from the Source
+## Contributing to Ballerina
+
+As an open source project, Ballerina welcomes contributions from the community.
+
+All Ballerina Standard Library **bug reports/feature requests** are maintained in [ballerina-standard-library](https://github.com/ballerina-platform/ballerina-standard-library) repository. 
+
+You can also check for [open issues](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fsql) that interest you. We look forward to receiving your contributions. Do not forget to suggest `module/sql` label for issues related to SQL module.
```

Comment:
```suggestion
You can also check for [open issues](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fsql) that interest you. We look forward to receiving your contributions. Do not forget to add the `module/sql` label for issues related to the SQL module.
```

Reply:
@praneesha Only committers can add the labels. Others can only suggest in the issues description